Atlassian.Jira/CustomField.cs

```diff
@@ -23,26 +23,32 @@ public CustomField(RemoteField remoteField)
 
         internal RemoteField RemoteField
         {
-            get
-            {
-                return this._remoteField;
-            }
+            get { return this._remoteField; }
         }
 
         public string CustomType
         {
-            get
-            {
-                return _remoteField.Schema?.Custom;
-            }
+            get { return _remoteField.Schema?.Custom; }
         }
 
         public string CustomIdentifier
         {
-            get
-            {
-                return _remoteField.Schema?.CustomId;
-            }
+            get { return _remoteField.Schema?.CustomId; }
+        }
+
+        public string Type
+        {
+            get { return _remoteField.Schema?.Type; }
+        }
+
+        public string ItemsType
+        {
+            get { return _remoteField.Schema?.Items; }
+        }
+
+        public bool IsRequired
+        {
+            get { return _remoteField.IsRequired; }
         }

```

Comment:
nit: I know it was like that before your change but all of this could be expression body i.e.:
`public bool IsRequired => _remoteField.IsRequired;`